src/views/catalog/CreateFromInstanceTypes/components/SelectInstanceTypeSection/components/RedHatSeriesMenuCard/RedHatSeriesMenuCard.tsx

```diff
@@ -69,31 +69,32 @@ const RedHatSeriesMenuCard: FC<RedHatSeriesMenuCardProps> = ({
       isVisible={isMenuExpanded}
       popperMatchesTriggerWidth={false}
       trigger={
-        <MenuToggle
-          id="series-card-toggle"
-          onClick={(event) => onMenuToggle(event, seriesName)}
-          isExpanded={isMenuExpanded}
-          variant="plain"
-          className={classNames(
-            'instance-type-series-menu-card__toggle-container',
-            isSelectedMenu && 'selected',
-          )}
-        >
-          <Card className="instance-type-series-menu-card__toggle-card">
-            <div className="instance-type-series-menu-card__card-icon">
-              <Icon />
-            </div>
-            <CardBody>
-              <div className="instance-type-series-menu-card__card-title">{seriesTitle}</div>
-              <div className="instance-type-series-menu-card__card-toggle-text">
-                {seriesLabel || classAnnotation} <AngleDownIcon />
+        <div className="instance-type-series-menu-card">
```

Comment:
Hmm, what is the reason of adding this extra div with this class here? I don't see any reason for it... 
The change you've added `color: var(--pf-global--Color--100);` is applicable only for "instance-type-series-menu-card__toggle-card" class.

Reply:
I made a lot of changes with css in this PR, probably just forgot to remove this, thanks :)